packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -96,13 +96,14 @@ export function computeDefaults<
   S extends StrictRJSFSchema = RJSFSchema
 >(
   validator: ValidatorType<T, S>,
-  schema: S,
+  rawSchema: S,
   parentDefaults?: T,
   rootSchema: S = {} as S,
   rawFormData?: T,
   includeUndefinedValues: boolean | "excludeObjectChildren" = false
 ): T | T[] | undefined {
   const formData = isObject(rawFormData) ? rawFormData : {};
+  let schema: S = isObject(rawSchema) ? rawSchema : ({} as S);
```

Comment:
We have a test that passes an invalid schema and expects a specific error message. The test fails without this change because a string ends up getting passed here as `rawSchema`, and we don't get to the error case that the test does expect.

The best fix would be to actually validate the schema before we get this far, but that would be a large change that would require a lot of effort.